src/pivot-table/components/cells/ColumnAdjusterWrapper.tsx

```diff
@@ -32,7 +32,7 @@ const ColumnAdjusterWrapper = ({
   const { isActive } = useSelectionsContext();
   const styleService = useStyleContext();
   const [, forceRerender] = useState({});
-  const shouldRender = cellInfo.canBeResized && !!interactions.active && !isActive;
```

Comment:
We still need this. This component handles all the logic for whether it should be rendered or not. After this change, all dimensionValue and header cells have a column adjuster

Reply:
Fixed